build-tests/sparo-outputs/README.md

```diff
@@ -0,0 +1,43 @@
+# Sparo
```

Comment:
Let's write a simple README.md explaining what this project does and how to run it. Think from the perspective of a stranger who is making their first OSS contribution to Sparo.